src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1041,6 +1041,7 @@ void cbmc_parse_optionst::help()
     "\n"
     "Backend options:\n"
     " --object-bits n              number of bits used for object addresses\n"
+    " --malloc-may-fail            enable failures for malloc calls\n"
```

Comment:
:bulb: I think it'd be better to be a little more explicit. 

```
allow malloc calls to return a null pointer
```

Reply:
Changed.